lib/response_handler.rb

```diff
@@ -29,27 +26,50 @@ def search_word(path)
     end
   end
 
+  def clear_count
+    reset_count
+  end
+
+  def hello
+    @hello_count += 1
+    "<p>Hello, World! (#{hello_count - 1})</p>"
+  end
+
+  def datetime
+    datetime = Time.now.strftime("%I:%M%p on %A, %B %d, %Y")
```

Comment:
Do not use `Time.now.strftime` without zone. Use one of `Time.zone.now.strftime`, `Time.now.strftime.current`, `Time.now.strftime.in_time_zone`, `Time.now.strftime.utc`, `Time.now.strftime.getlocal`, `Time.now.strftime.iso8601`, `Time.now.strftime.jisx0301`, `Time.now.strftime.rfc3339`, `Time.now.strftime.to_i`, `Time.now.strftime.to_f` instead.